src/pages/fi-account-groups/Windows/AccountGroupsWindow.js

```diff
@@ -0,0 +1,29 @@
+import Window from 'src/components/Shared/Window'
+import AccountGroupsForm from '../forms/AccountGroupsForm'
+
+const AccountGroupsWindow = ({
+  onClose,
+  labels,
+  maxAccess,
+  recordId
+}) => {
+  
+  return (
+    <Window
+      id='AccountGroupsWindow'
+      Title={labels.accountGroups}
```

Comment:
the title of the window should be "Account Group" because we're adding 1 account group with each submit